scripts/deploy.sh

```diff
@@ -57,8 +57,12 @@ elif [ "$STUDIO" ]; then
     echo "Deploying on graph studio"
     yarn graph deploy --studio "$SUBGRAPH_NAME" --deploy-key "$DEPLOY_KEY"
 elif [ "$GOLDSKY" ]; then
-    echo "Deploying on Goldsky indexer"
-    goldsky subgraph deploy "$SUBGRAPH_NAME/latest" --path . --token "$API_KEY"
+    echo "Deploying $SUBGRAPH_NAME on Goldsky indexer"
+    echo "Existing versions:"
+    goldsky subgraph list "$SUBGRAPH_NAME" --summary --token "$GOLDSKY_API_KEY"
+    echo "Enter the version number"
+    read VERSION
+    goldsky subgraph deploy "$SUBGRAPH_NAME/$VERSION" --path . --token "$GOLDSKY_API_KEY"
```

Comment:
Goldsky handles versions differently from The Graph; we need to specify the version in the name, and therefore, we cannot deploy the same subgraph name multiple times. Instead, we need to increment the version. In The Graph, this is handled automatically.
These changes in the script address these differences.